plans/e2e/multi-bluechi-agents.fmf

```diff
@@ -0,0 +1,39 @@
+summary: multi-bluechi-agents - QM Interconnect through bluechi
```

Comment:
isnt it a duplication of of plans ffi ? 
maybe you could use one plan for prepare could go under plans? and then the other plans for ffi and e2e will have their own difference in prepare step?

Reply:
I tried to extract the common parts of `ffi.fmf` and `tier-0.fmf ` to `plans/main.fmf ` , but main.fmf will affect the `multi-bluechi-agents.fmf` and `tier-1.fmf`. This makes me feel a bit confusing.